kip-0017.md

```diff
@@ -0,0 +1,489 @@
+This contains information on which of the requested items it supports.
+
+### Example Pairing Proposal Request and Settlement Response
+
+```jsonc
+// Proposal Request
+{
+  // ... other properties from the WalletConnect Proposal Request
+  "requiredNamespaces": {
+    "kadena": {
+      "chains": ["kadena:mainnet01", "kadena:testnet04", "kadena:development"],
+      "methods": [
+        "kadena_getAccounts_v1",
+        "kadena_sign_v1",
+        "kadena_quicksign_v1"
+      ],
+      "events": []
+    }
+  }
+}
+```
+
+```jsonc
+// Settlement Response
+{
+  // ... other properties from the WalletConnect Settlement Response
+  "namespaces": {
+    "kadena": {
+      "accounts": [
+        "kadena:mainnet01:38298612cc2d5e841a232bd08413aa5304f9ef3251575ee182345abc3807dd89",
+        "kadena:testnet04:38298612cc2d5e841a232bd08413aa5304f9ef3251575ee182345abc3807dd89",
+        "kadena:testnet04:22ddc64851718e9d41d98b0f33d5e328ae5bbbbd97aed9885adac0f2d070ff9c"
+      ],
+      "methods": [
+        "kadena_getAccounts_v1",
+        "kadena_sign_v1",
+        "kadena_quicksign_v1"
+      ],
+      "events": []
+    }
+  }
+}
+```
+
+### WalletConnect Proposal Request
+
+WalletConnect dictates a `requiredNamespaces` property that contains:
+
+1. `requiredNamespaces`: `Object`\
+    1.1. `chains`: `Array<string>` - an array of [Chain IDs](#wallet-connect-chain-ids) requested.\
+    1.2. `methods`: `Array<string>` - an array of [Methods](#walletconnect-methods) requested.\
+    1.3. `events`: `Array<string>` - an array of [Events](#walletconnect-events) requested.
+
+### WalletConnect Settlement Response
+
+The response for the Proposal contains a `namespaces` object with the following properties:
+
+1. `namespaces`: `Object`\
+    1.1. `accounts`: `Array<string>` - an array of [Accounts](#wallet-connect-accounts) availble for signing.\
+    1.2. `methods`: `Array<string>` - an array of [Methods](#walletconnect-methods) available.\
+    1.3. `events`: `Array<string>` - an array of [Events](#walletconnect-events) available.
+
+## Wallet Connect Chain IDs
+
+Chain IDs are Wallet Connect’s way of identifying specific blockchains within an ecosystem.
+
+They are defined by the [CAIP-2 standard](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) and are formatted as:
+
+```
+    <namespace> + ":" + <reference>
+```
+- `<namespace>`: `string` - always defined as "kadena".
+- `<reference>`: `string` - a unique network ID within the blockchain ecosystem (e.g. `testnet04`, `mainnet01`).
+
+These are valid Chain IDs:
+
+- `kadena:mainnet01` - Kadena mainnet
+- `kadena:testnet04` - Current Kadena testnet
+- `kadena:development` - Kadena devnet
+
+> **NB**: `Chain IDs`, as defined here, are semantically specific to
+> WalletConnect and are not equivalent to a Kadena "Chain ID" (e.g. chain 1).
+> They are more closely related to Kadena's `networkId` field.
```

Comment:
Why aren't they called "Network IDs", then?

Reply:
In this context the name "Chain ID" is defined by the CAIP-2 standard which the Wallet Connect specification adheres to. We're unable to change that name, but merely added this note to avoid confusion with Kadena's Chain ID.

Reply:
We want to use the naming that WalletConnect uses in places where it's dictated by WalletConnect